config/locales/en/prison_views.yml

```diff
@@ -25,9 +26,28 @@ en:
       message_create_error: Your message couldn't be saved
       message_created: Message created
       no_cancellation_reason: Please provide a cancellation reason
-      process_required: Please either reject or accept the booking
+      process_required_html: |
+        <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
+          This visit wasn't booked.
+        </h1>
+        <p>Please either reject or accept the booking</p>
       at_least_one_estate: You must select at least one estate
       invalid_estate_selection: You don't access to these estates
+      nomis_validation_error_html: |
+        <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
+          This visit wasn't booked.
+        </h1>
+        <p>Choose a rejection reason to reject the visit or <a href="#nomis-opt-out">book this visit anyway</a>.</p>
```

Comment:
@kayesoutham @kyliehavelock I'm wondering if we should tweak this validation error message. If the prisoner has a restriction, such as a Closed restriction, staff won't really know what validation failed. I think it would be helpful to explain that this validation error can happen when the offender has restrictions and to check in Nomis. What do you think?